Revamp tool pages (#7700)
ballerina-platform/ballerina-dev-website: swan-lake/integration-tools/asyncapi-tool.md

The Ballerina AsyncAPI tool makes it easy for you to start the development of an event API documented in an AsyncAPI contract in Ballerina by generating a Ballerina service and listener skeletons.

## Set up the prerequisites

To run this tutorial, you need the following prerequisites:

1. [Ballerina 2202.1.0 (Swan Lake)](/downloads/) or greater
2. A text editor
  >**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the 
  <a href="https://wso2.com/ballerina/vscode/docs/" target="_blank">Ballerina extension</a> installed.

> 

Author: We need not specifically mention Ballerina and VS Code on every Learn page as it will be redundant content.

| `-i, --input`  | The `asyncapi-contract-path` parameter specifies the path of the AsyncAPI contract file (e.g., `my-api.yaml` or `my-api.json`).                                                                                                                                                                                                                                                                                                                               | mandatory           |
| `-o, --output`     | The Ballerina files get generated at the same location from which the `bal asyncapi` command is executed. Optionally, you can point to another directory location by using this flag.                                                                                                                                           | optional           |

> This command generates a Ballerina source (i.e., the four Ballerina files below) from the given AsyncAPI definition file.

Reviewer: ```suggestion
This command generates Ballerina service and listener skeletons (i.e., the four Ballerina files below) from the given AsyncAPI definition file.
```
